contents/handbook/growth/marketing/components.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Content components
+sidebar: Handbook
+showTitle: true
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+This page details various code components you can add to articles. 
+
+## Call to action
+
+Adding `<ArrayCTA />` to any article will add this simple CTA:
+
+<ArrayCTA />
+
+Don't overuse it, but it's useful for high intent pages, like comparisons.
```

Comment:
Maybe a tad more specific - I assume you mean once or twice is appropriate? (Just as traditional marketing would go 'yeah 4 CTAs is fine' but I don't think we would)

Reply:
Eh, tbh, I don't think we need to be super specific, otherwise it's just a bunch of rules where common sense can prevail.